src/widgets/cs-program-editor.jsx

```diff
@@ -181,9 +181,12 @@ var CSProgramEditor = React.createClass({
                 });
             })
             .fail((jqxhr, textStatus, error) => {
+                /* eslint-disable no-console */
                 console.error("Error retrieving scratchpad info for " +
```

Comment:
Not really sure this is a useful error, you could find the same thing in the network panel. I'd vote for nixing this.